.github/workflows/test-application.yaml

```diff
@@ -188,11 +189,11 @@ jobs:
             fail-fast: false
             matrix:
                 include:
-                    - php-version: '8.1'
-                      node-version: '16'
-                      npm-version: '6'
+                    - php-version: '8.3'
+                      node-version: '20'
+                      npm-version: '10'
                       mysql-version: '8.0'
-                      php-extensions: 'ctype, iconv, intl, mysql, pdo_mysql, php_fileinfo, imagick'
+                      php-extensions: 'ctype, iconv, intl, mysql, pdo_mysql, php_fileinfo, gd, sodium'
```

Comment:
![Bildschirmfoto 2024-03-15 um 10 31 41](https://github.com/sulu/skeleton/assets/1698337/01197d2e-fa69-412b-87df-26d426d4267b)

Looks like Imagick on PHP 8.3 is not yet supported for windows.

The `sodium` PHP Extension seems not longer be part of setup-php of winodws. So we need to enable it.